sdk/python/tests/compiler/compiler_tests.py

```diff
@@ -711,6 +717,29 @@ def some_pipeline():
       container = template.get('container', None)
       if container:
         self.assertEqual(template['retryStrategy']['limit'], 5)
+        
+  def test_oob_component_label(self):
+    gcs_download_op = components.load_component_from_url(
+        _TEST_GCS_DOWNLOAD_COMPONENT_URL)
+    
+    @dsl.pipeline(name='some_pipeline')
+    def some_pipeline():
+      _download_task = gcs_download_op('gs://some_bucket/some_dir/some_file')
+      
+    workflow_dict = compiler.Compiler()._compile(some_pipeline)
+    
+    found_download_task = False
+    for template in workflow_dict['spec']['templates']:
+      if template.get('container', None):
+        found_download_task = True
+        self.assertEqual(
+            template['metadata']['labels'][
+                'pipelines.kubeflow.org/component_origin_path'],
```

Comment:
I think it's better to import and use `_COMPONENT_PATH_LABEL_KEY`

Reply:
Done.